lib/whois/parsers/whois.rotld.ro.rb

```diff
@@ -45,13 +45,20 @@ class WhoisRotldRo < Base
         !available?
       end
 
+      property_supported :expires_on do
```

Comment:
Can you please keep the property definition sorted as in the previous version.

Reply:
I've pushed update